README.md

```diff
@@ -1,17 +1,24 @@
 # Compose LazyList/Grid reorder
 [![Latest release](https://img.shields.io/github/v/release/aclassen/ComposeReorderable?color=brightgreen&label=latest%20release)](https://github.com/aclassen/ComposeReorderable/releases/latest)
 
-A Jetpack Compose (Android + Desktop) modifier enabling reordering by drag and drop in a LazyList and LazyGrid.
+A Jetpack Compose modifier enabling reordering by drag and drop in a LazyList and LazyGrid.
 
 ![Sample](readme/sample.gif)
 
-## Download
+## WHOOP Fork
 
-```
-dependencies {
-    implementation("org.burnoutcrew.composereorderable:reorderable:<latest_version>")
-}
-```
+This library is forked from Andre Claßen's open source library
+[ComposeReorderable](https://github.com/aclassen/ComposeReorderable).
+
+We are using a custom WHOOP fork so that we can customize the implementation to suit our needs,
+as well as to mitigate risk in case the 3rd party library ever stops being maintained.
+
+The `main` branch is just a copy of the upstream library's `main` branch, while the `whoop` branch
+contains any whoop-specific code. When making whoop-specific changes, you should branch off from
+`whoop` into a new feature branch, and open a PR back onto the `whoop` branch.
```

Comment:
So a dev working on this would just never touch main? Also main is locked in place in this repo right? So if a bad actor pushes some malware into the main branch of the base repo and doesn't update the version or anything else we wouldn't pull that in?

Reply:
Yes, I locked `main` for anything besides pulling in commits from the upstream library. Tried to keep the same setup we have as the Paparazzi fork, but lmk if I missed anything here @ncipollo